src/lev-dashboard/DashboardContent.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styled from 'styled-components';
+import DashboardCounter from "./DashboardCounter";
+
+const CountDiv = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: row;
+`;
+
+const REFRESH = 10 * 1000;
+const FRAMES = 4;
+const DURATION = 1500;
```

Comment:
Could these be props? (With those numbers as the defaults.)

Reply:
They could be - BUT, they'd have NO meaning for the server side rendering code.

Reply:
I think that's probably fine.

Reply:
Let's leave this for another day.